task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -295,6 +295,47 @@ def test_run_raises_base_exception(time_machine, mocked_parse, make_ti_context):
         )
 
 
+def test_run_raises_system_exit(time_machine, mocked_parse, make_ti_context):
+    """Test running a basic task that exits with SystemExit exception."""
+    from airflow.providers.standard.operators.python import PythonOperator
+
+    task = PythonOperator(
+        task_id="system_exit_task",
+        python_callable=lambda: exit(10),
+    )
+
+    what = StartupDetails(
+        ti=TaskInstance(
+            id=uuid7(),
+            task_id="system_exit_task",
+            dag_id="basic_dag_system_exit",
+            run_id="c",
+            try_number=1,
+        ),
+        file="",
+        requests_fd=0,
+        ti_context=make_ti_context(),
+    )
+
+    ti = mocked_parse(what, "basic_dag_system_exit", task)
+
+    instant = timezone.datetime(2024, 12, 3, 10, 0)
+    time_machine.move_to(instant, tick=False)
+
+    with mock.patch(
+        "airflow.sdk.execution_time.task_runner.SUPERVISOR_COMMS", create=True
+    ) as mock_supervisor_comms:
```

Comment:
Just use the mock_supervisor_comms fixture

Reply:
Alright, yeah, didn't realise it was added in #45245